# rover_out.msg
# used to pass variables from the Rover5 platform to the ROS environment  

Header header
uint16 lSpeedAct
uint16 rSpeedAct
uint16 pingDist
int16 imuXAccel
int16 imuYAccel
int16 imuZAccel
int32 lPos
int32 rPos

